app/src/main/java/com/example/snookerapi/retrofit/SnookerApi.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.snookerapi.retrofit
+
+import com.example.snookerapi.model.Player
+import com.example.snookerapi.model.PlayerProfile
+import retrofit2.Call
+import retrofit2.http.*
+
+interface SnookerApi {
+
+   /* @Headers("X-Requested-By: ArtemMinsk")*/
+    @GET("?t=10&st=p")
+    //@GET("?t=10&st=p&s=2021")
+    fun getPlayers(
+       @Query("s") s: Int
+    ): Call<List<Player>>
+
+    @GET(".")
```

Comment:
не знаю как работает с точкой, но уже тогда можно просто без параметров
```suggestion
    @GET
```